src/boot/efi/meson.build

```diff
@@ -54,14 +63,17 @@ if efi_ld == 'auto'
         endif
 endif
 
+efi_multilib = '/usr/lib' / run_command(
+        cc.cmd_array(),
+        '-print-multi-os-directory',
+        get_option('efi-cflags'),
+        check: false
+).stdout().strip()
```

Comment:
Hmm, doing `run_command` as an operand of `/` seems a bit too much. Maybe split that into two steps?